style.css

```diff
@@ -395,3 +429,407 @@ textarea:focus {
   gap: 10px;
   padding: 0;
 }
+
+@media (min-width: 990px) {
+  .mob-img {
+    display: none;
+  }
+
+  .desk-li {
+    display: flex;
+    justify-content: start;
+    list-style: none;
+    gap: 15px;
+    margin-right: 156px;
+  }
+
+  .desk-head {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    margin-right: 54px;
+  }
+
+  .name-me {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 20px;
+    text-align: center;
+    padding: 12px;
+    margin-left: 120px;
+  }
+
+  .desk-img {
+    color: #67798e;
+    width: 20px;
+    height: inherit;
+    border: solid 2px;
+    border-radius: 4px;
+    border-color: #fff;
+  }
+}
+
+@media (max-width: 990px) {
+  .desk-head {
+    display: none;
+  }
+}
+
+@media (min-width: 990px) {
+  .heading {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: start;
+    gap: 89px;
+    background-image: url(icons/Illustration-header@2x.png);
+    background-repeat: no-repeat;
+    background-position-y: 0;
+  }
+
```

Comment:
- [ ] Nice job 👏  👏  on working on your style however you still need to work on some styling and make your design look like the template one.
- Kindly work on the position and size of the background of your heading section so that it may look like the original template. check the difference here 👇 

Your design | The template
----|---
![image](https://user-images.githubusercontent.com/65068771/149508738-c681922e-28a7-40e7-80f7-84d3f06a12b7.png) |![image](https://user-images.githubusercontent.com/65068771/149508973-8619a29c-196c-48a2-a5a9-8d15d124283e.png)